### ISSUE TITLE
legislators.json is out-of-date

### ISSUE THREAD

**Issue by mfb** [EFForg/congress-zip-plus-four#35]
legislators.json is missing some legislators e.g. K000388 D000625 L000585

**Comment by j-ro** [comment]
Indeed, probably never been updated. Feel free to submit a PR with missing data!

**Comment by mfb** [comment]
Fingers crossed that there is a script to auto-generate this file..?  :)

**Comment by j-ro** [comment]
At one point there may have been -- it essentially went through each member's website and looked for addresses of their local offices I think. Anyway, we have some probably more up to date data here:

https://github.com/j-ro/congress-forms-data
